src/util.cpp

```diff
@@ -72,43 +73,7 @@ timer wctime() {
     return time;
 }
 
-// All are base-0
```

Comment:
If possible, small preference to keep those in the .cpp and to use a single "integer" vector type.